portfolio/src/main/webapp/script.js

```diff
@@ -74,3 +109,31 @@ function addRandomQuote() {
   quoteContainer.innerText = quote;
 }
 
+/**
+ * Add a chart to the page.
+ */
+function drawChart() {
+  fetch('/wordtree').then((response) => response.text()).then((sentenceJSONList) => {
+    sentenceListObj = JSON.parse(sentenceJSONList);
+    const dataList = [['Sentences']].concat(sentenceListObj);
+    console.log(dataList);
+    var data = new google.visualization.arrayToDataTable(dataList);
```

Comment:
Use const/let instead of var throughout. The two are an ES6 replacement for var.